SharkCage/CageConfigurator/CageConfiguratorForm.Designer.cs

```diff
@@ -185,18 +189,34 @@ private void InitializeComponent()
             this.videoSources.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
             this.videoSources.FormattingEnabled = true;
             this.videoSources.ItemHeight = 13;
-            this.videoSources.Location = new System.Drawing.Point(12, 383);
+            this.videoSources.Location = new System.Drawing.Point(12, 398);
             this.videoSources.Name = "videoSources";
             this.videoSources.Size = new System.Drawing.Size(256, 21);
             this.videoSources.TabIndex = 7;
             this.videoSources.TabStop = false;
             this.videoSources.Visible = false;
             // 
+            // restrictExitButton
+            // 
+            this.restrictExitButton.Location = new System.Drawing.Point(12, 82);
+            this.restrictExitButton.Name = "restrictExitButton";
+            this.restrictExitButton.Size = new System.Drawing.Size(173, 17);
+            this.restrictExitButton.TabIndex = 0;
+            this.restrictExitButton.Text = "Restrict exiting to Cage Labeler";
+            this.restrictExitTooltip.SetToolTip(this.restrictExitButton, resources.GetString("restrictExitButton.ToolTip"));
```

Comment:
Is there a way to specify how long the tooltip is displayed? It is awesome that it is possible to get more informations about the feature but it is so much text to read (but good text :+1:), that the tooltip is closing while reading the text. 
If there is no way (or it will took too much time) this is also acceptable.

Reply:
I changed to 30s (from 5s), its not possible to show it much longer